specs/_features/maxeb_increase/capella.py

```diff
@@ -1144,11 +1190,7 @@ def initiate_validator_exit(state: BeaconState, index: ValidatorIndex) -> None:
         return
 
     # Compute exit queue epoch
-    exit_epochs = [v.exit_epoch for v in state.validators if v.exit_epoch != FAR_FUTURE_EPOCH]
-    exit_queue_epoch = max(exit_epochs + [compute_activation_exit_epoch(get_current_epoch(state))])
-    exit_queue_churn = len([v for v in state.validators if v.exit_epoch == exit_queue_epoch])
-    if exit_queue_churn >= get_validator_churn_limit(state):
-        exit_queue_epoch += Epoch(1)
+    exit_queue_epoch = compute_exit_epoch_and_update_churn(state, state.balances[index])
```

Comment:
```suggestion
    exit_queue_epoch = compute_exit_epoch_and_update_churn(state, validator.effective_balance)
```

Pointed by @fradamt, `balances[index]` from the `effective_balance`, exit churn should rather be based on the latter.